On branch edburns-msft-jea-200-gh-906 Changes for XSD updates and unrelated copyediting. (#911)
jakartaee/platform: specification/src/main/asciidoc/platform/PreviousVersionDeploymentDescriptors.adoc

[appendix]
[[a3447]]
== Deployment Descriptors

This appendix describes Document Type Definitions (DTDs) and XML
schemas for Deployment Descriptors from current and previous versions
of the Java(TM) EE and Jakarta(TM) EE specifications.  All Jakarta EE
9 products are required to support the DTDs and schemas specified by
Jakarta EE 8, as well as the schemas specified in the current version
of the specification.  This ensures that applications written to the
previous version of the Jakarta EE specification can be deployed on
products supporting the current version of this specification.
Support for versions of DTDs and schemas prior to Jakarta EE 8 is
optional.  In addition, there are no restrictions on mixing versions
of supported deployment descriptors in a single application; any
combination of valid deployment descriptor versions must be supported.

> If a component specification is modified during an EE release cycle,
> and that newer version of the specification is included in the EE
> release cycle, any XSD files must have their version number and file
> names updated accordingly to match the version of the component
> specification included in the corresponding EE release.  This update
> must happen regardless whether or not there are normative changes to
> the XSD files.
> 

Reviewer: ```suggestion
If a newer version of a component specification is included in the Jakarta EE
release cycle, any XSD file used by that specification has its version number and file
name updated accordingly to match the version of the component
specification included in the corresponding EE release.  This update
happens whether or not there are normative changes to the XSD files.

```

Author: Fixed in 406fa5a.
